cfgov/unprocessed/css/molecules/social-media.less

```diff
@@ -36,62 +36,72 @@
 */
 
 .m-social-media {
-  display: inline-block;
-
-  // This could be wrapped in the `m-social-media__share` modifier,
-  // but it's ok on its own too.
-  &_heading {
-    .heading-5();
-    display: inline-block;
-    // 0.25em subtracted to compensate for inline spacing
-    padding-right: unit(@grid_gutter-width / @font-size - 0.25em, em);
-    margin-bottom: 0;
-    vertical-align: middle;
-  }
+  display: inline-flex;
+  flex-wrap: wrap;
+  flex-direction: column;
```

Comment:
Do we actually need `flex-direction: column`? Look at https://www.consumerfinance.gov/1071-rule/ vs http://localhost:8000/1071-rule/ and resize to mobile. The stacking behavior differs between the two, with this PR stacking sooner. With `flex-wrap: wrap` it'll stack when room runs out. If we want it stacking sooner that's fine, but it seems preferable to only stack when room runs out.